modules/social_features/social_activity/social_activity.tokens.inc

```diff
@@ -106,7 +111,28 @@ function social_activity_tokens($type, $tokens, array $data, array $options, Bub
               }
             }
           }
+          break;
 
+        case 'count_groups_per_node':
+          if (!$message->get('field_message_related_object')->isEmpty()) {
+            $target_type = $message->get('field_message_related_object')->getValue()[0]['target_type'];
+            $target_id = $message->get('field_message_related_object')->getValue()[0]['target_id'];
+
+            if ($entity = \Drupal::entityTypeManager()
+              ->getStorage($target_type)
+              ->load($target_id)) {
+              // For the moment we can count only for group_content and node entities.
+              $count = \Drupal::service('social_group.cross_posting')
+                ->countGroupsByGroupContentNode($entity);
+              // Recount as token renders in group context and we need to
+              // exclude the one.
+              $count = $count ? $count - 1 : $count;
+              // Build the string like "(+ 4 groups more)".
+              $replacements[$original] = $count
```

Comment:
I think we can use the format Plural here in order to ensure we render the string based on the Group count
e.g. + 1 more vs +2/3/4 more etc. That should help our customers in their translation efforts.